### ISSUE TITLE
Admin Panel not finding users and plugins

### ISSUE THREAD

**Issue by Phil1ooo** [WWBN/AVideo#1591]
Hello Daniel
I have installed your script and also purchased a couple of plugins and there is a few issues with it all working together. 
Firstly I am using this on a dedicated server with FFmpeg and nginx plus all others installed on a centos7 also using cPanel/WHM/Cloudlinux. As this is a self managed server I have access to everything.

1) Having install this scrip so far works but there is an issue with the admin dashboard not displaying users, it registers users without issues and lets them log in but doesn't display the users (including the admin) in the admin dashboard.
2) I feel this is related to 1) issue as when wanting to test the streaming it says I am not logged in .
3) After many days of issues in installing the Encoder as nothing we tried worked I finally figured out (and I hope I am correct here) figured out that it had to be installed in the plugins folder. NOWHERE has 2 of use spending 2 days reading every possible document do we find anywhere that said any of your purchased addons had to be installed via the admin plugin or even be uploaded into the plugin folder. Why is that?
4) I purchased the Encoder and Footer Customize addon and again (same issue as above "anyway") now installed via the addmin still displays as needing to be purchased instead of providing editing options.
5) This also applies to the LiveUser addon as well.

Basically the script has trouble reading addons and finding users.

I like this script and as I get one step forward means other addons get included/added as I want a full operational working system.
![streaming](https://user-images.githubusercontent.com/384952/54960711-efa6c300-4fb1-11e9-8690-bdf6f8dc40ea.png)

**Comment by Phil1ooo** [comment]
Daniel's reply

Hi

We strongly recommend you to use Ubuntu instead of Centos, especially if you are using a Cpanel.

The behaviors on Centos are unpredictable. you need a Ubuntu without any kind of control panel

**Comment by Phil1ooo** [comment]
HI Daniel. This is not going to happen as this a running server, plus as you said it's a linux so no reason why it shouldn't work. 
Also I am sure that the issues involved here have nothing to do with whether it's a ubuntu or centos as they do work and it is your admin panel not finding them correctly.

Other than these minor issue everything else works, uploading videos, encoding, using account etc etc ,,

Can you find a fix for the admin panel not doing what it technically suppose to be doing and that is finding plugins and users. It's probably nothing more than a misconfiged path somewhere which you as the designer would know.

I have purchased these plugins, and will purchase more but this needs to work and work on centos which as I stated does.

Can you help Please.

**Comment by Phil1ooo** [comment]
The above 3 posts are copies from the Disqus. I choose to move it to here as most other responses made on disqus by Daniel where to create a ticket here so here I am.
Also it could help others within here in the github forum.

Hope you understand Daniel.

Cheers

**Comment by DanielnetoDotCom** [comment]
@Phil1ooo thanks for moving the thread to here.

Actually, you can make it work on Centos, I just do not support it. I have bad experiences when I try to compile some libs, especially the Livestream ones.

Usually, Ubuntu installations are much more easy to complete.

I also can see you enable the experimental webcam support, what I also do not recommend, it is not working good. to stream try this way https://tutorials.youphptube.com/video/youphptube-video-stream-with-obs-and-vmix

Also check your nginx stats page to see how it looks like, check our stats how it looks like https://youphptube.com:444/stat

**Comment by Phil1ooo** [comment]
This issue is now over as Daniel has decided to not support centos when his product requesting I only use ubuntu.

He has kindly refunded my purchased and the software will be deleted now and going to another script that is better supported.

I'd like to add that this product was one of the better ones I found and it's a real shame Daniel has chosen not to support it. Considering all the others like it do support centos I like to suggest that Daniel rethink his policies in thinking ubuntu is the bees knees of OS's as it's by far the least used in the business hosting world and thus destroying his own potentials in his product being used by more video site owners on the internet.

Wish you luck and if you ever decide to bring yourself up to date with today's most used technology please let me know as I may just start a second site in the same lines soon .
Cheers

**Comment by AncientTom** [comment]
> This issue is now over as Daniel has decided to not support Centos when his product requesting I only use Ubuntu.
> 
> He has kindly refunded my purchased and the software will be deleted now and going to another script that is better supported.
> 
> I'd like to add that this product was one of the better ones I found and it's a real shame Daniel has chosen not to support it. Considering all the others like it do support Centos I like to suggest that Daniel rethink his policies in thinking Ubuntu is the bees knees of OS's as it's by far the least used in the business hosting world and thus destroying his own potentials in his product being used by more video site owners on the internet.
> 
> Wish you luck and if you ever decide to bring yourself up to date with today's most used technology please let me know as I may just start a second site in the same lines soon .
> Cheers

It's to bad that the creator would determine that Centos is not a good platform. I use Ubuntu 18.04 LTS on my Desktop. This is what the Ubuntu flavor was originally developed for integrating the best of graphics interfaces into it. I also administrate my own leased server with OVH in Canada which is running Centos7 which is a Redhat flavor minus the proprietary ingredients. I use Centos because it is a designed server OS unlike Ubuntu without the X server interfaces. I have all my Internet services being served by this machine including my primary DNS service. It's a real bummer to be told that if I want to use a fine piece of software, I have to use Ubuntu. It might as well be said that I have to run it in Windows, heaven forbid.

It seems to me that with my attempted installation, which looks like everything installed well in Centos, there are a few sub-directories that weren't created, one of which was the user directory. when I open the AVideo home page it doesn't have anything populated on it. I select "sign in" and get "The requested URL /AVideo/user/ was not found on this server. I created the directory manually with apache permissions and now get "You don't have permission to access /AVideo/user/ on this server".

I can't help thinking that this should be a reasonably easy fix if I knew what all was missing and not reallyl a Centos issue.

**Comment by DanielnetoDotCom** [comment]
It is possible to install on Centos but will be much easier if you try to install on Ubuntu.

I just cannot support Centos installation. (Specialty because of cPanel). I do not use it for myself, so I have no idea what is missing.

**Comment by AncientTom** [comment]
> It is possible to install on Centos but will be much easier if you try to install on Ubuntu.
> 
> I just cannot support Centos installation. (Specialty because of cPanel). I do not use it for myself, so I have no idea what is missing.

Daniel; I really applaud the prompt way that you have been responding to support your product, unlike most other software support forums.

As I tried to point out in my last post is that my server is already committed to Centos and all my services are running on it. To switch OSs  after deployment of the server would mean taking my services off line for a month while i set up a Ubuntu server and re-install all my services including Postfix Mail Server which is a real headache to have to set up from scratch.

I have to say, my specific problem may not be a compatibility issue between your project and Centos. As I said earlier, I do use Ubuntu as my primary desktop so because of my familiarity with Ubuntu and all the other software that is already installed on my remote server, I probably have already installed all the dependencies of AVideo onto my machine. 

OVH originally installed Centos7 onto my machine and isolated the root file system, "/", by placing it on a separate partition of 20 gigabytes. This morning I tried to send you an email that failed because of a full file system. I don't know what OVH was thinking by only allotting 20 Gig for root file space.

Today, I will be trying to move over some of my installed applications to the secondary partition to free up root FS space and try re-installing AVideo again. I'll post results here if my Centos install was successful.
